pkg/option/config.go

```diff
@@ -45,6 +45,8 @@ import (
 	"github.com/cilium/cilium/pkg/mac"
 	"github.com/cilium/cilium/pkg/time"
 	"github.com/cilium/cilium/pkg/version"
+
+	"github.com/cilium/ebpf"
```

Comment:
nit: I think this import clause should be part of the 2nd import group (external non-stdlib dependencies) above. We generally try to stick to 3 import groups:

1. Go stdlib
2. External non-stdlib dependencies
3. Own module (github.com/cilium/cilium)

Reply:
thanks, fixed now!